File: app/src/main/java/dev/ragnarok/fenrir/model/Lang.java
package dev.ragnarok.fenrir.model;

import androidx.annotation.IntDef;

import java.lang.annotation.Retention;
import java.lang.annotation.RetentionPolicy;


@IntDef({Lang.DEFAULT, Lang.ENGLISH, Lang.RUSSIA})
@Retention(RetentionPolicy.SOURCE)
public @interface Lang {
    int DEFAULT = 0;
    int ENGLISH = 1;
    int RUSSIA = 2;
}

File: app/src/main/kotlin/dev/ragnarok/fenrir/model/Lang.kt
package dev.ragnarok.fenrir.model

import androidx.annotation.IntDef

@IntDef(Lang.DEFAULT, Lang.ENGLISH, Lang.RUSSIA)
@Retention(AnnotationRetention.SOURCE)
annotation class Lang {
    companion object {
        const val DEFAULT = 0
        const val ENGLISH = 1
        const val RUSSIA = 2
    }
}